From a5dcecfb817b256b0e36f3c48605f79e35dd3e57 Mon Sep 17 00:00:00 2001
From: Andy Green <andy@openmoko.com>
Date: Thu, 13 Nov 2008 20:28:59 +0000
Subject: [PATCH] fix-s3c2410-usb-controller-mode-change-always-EINVAL.patch

We always return -EINVAL when changing s3c2410 USB host and device modes
due to lack of { }.

Signed-off-by: Andy Green <andy@openmoko.com>
---
 drivers/usb/host/ohci-s3c2410.c |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/drivers/usb/host/ohci-s3c2410.c b/drivers/usb/host/ohci-s3c2410.c
index 68f5689..1c41202 100644
--- a/drivers/usb/host/ohci-s3c2410.c
+++ b/drivers/usb/host/ohci-s3c2410.c
@@ -335,9 +335,11 @@ static ssize_t set_usb_mode(struct device *dev, struct device_attribute *attr,
 		printk("s3c2410: changing usb to device\n");
 		s3c2410_modify_misccr(S3C2410_MISCCR_USBHOST, 0);
 		s3c2410_gpio_setpin(S3C2410_GPB9, 1);
-	} else
+	} else {
 		printk("s3c2410: unknown mode\n");
 		return -EINVAL;
+	}
+
 	return count;
 }
 
-- 
1.5.6.5

